webui/package-lock.json

```diff
@@ -12251,13 +12288,14 @@
       }
     },
     "node_modules/katex": {
-      "version": "0.16.10",
-      "resolved": "https://registry.npmjs.org/katex/-/katex-0.16.10.tgz",
-      "integrity": "sha512-ZiqaC04tp2O5utMsl2TEZTXxa6WSC4yo0fv5ML++D3QZv/vx2Mct0mTlRx3O+uUkjfuAgOkzsCmq5MiUEsDDdA==",
+      "version": "0.16.21",
```

Comment:
This is a katex version bump.  I think we need to test it manually by looking at how some things are rendered.  IIRC the easiest way to exercise Katex is to look at some LaTeX-style mathematics markup ($\sum_{n=1}^{\infty} \frac{1}{n^2} = \frac{\pi^2}{6}$, which comes from saying `$\sum_{n=1}^{\infty} \frac{1}{n^2} = \frac{\pi^2}{6}$`) in an ipynb notebook.

I think @Annaseli is correct that _we can mostly automate this_, if we have a Playwright test that looks at images pixel-wise.

Reply:
Works - thanks!
<img width="435" alt="image" src="https://github.com/user-attachments/assets/30688d4a-4eb5-4d83-8b0a-37ba927393ef" />

Reply:
Download sample notebook that showcase different equations.
This one is part of the following cell, where it seems that they added commas:

```
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\\begin{aligned}\n",
    "\\nabla \\times \\vec{\\mathbf{B}} -\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{E}}}{\\partial t} & = \\frac{4\\pi}{c}\\vec{\\mathbf{j}} \\\\\n",
    "\\nabla \\cdot \\vec{\\mathbf{E}} & = 4 \\pi \\rho \\\\\n",
    "\\nabla \\times \\vec{\\mathbf{E}}\\, +\\, \\frac1c\\, \\frac{\\partial\\vec{\\mathbf{B}}}{\\partial t} & = \\vec{\\mathbf{0}} \\\\\n",
    "\\nabla \\cdot \\vec{\\mathbf{B}} & = 0\n",
    "\\end{aligned}"
   ]
  },
```